lib/plugins/aws/customResources/generateZip.js

```diff
@@ -5,20 +5,19 @@ const fse = require('fs-extra');
 const getTmpDirPath = require('../../../utils/fs/getTmpDirPath');
 const createZipFile = require('../../../utils/fs/createZipFile');
 const ensureArtifact = require('../../../utils/ensureArtifact');
+const safeMoveFile = require('../../../utils/fs/safeMoveFile');
 
 const srcDirPath = path.join(__dirname, 'resources');
 
-const artifactName = 'custom-resources.zip';
-
-module.exports = async () => {
+module.exports = async (artifactName) => {
```

Comment:
What's the reasoning behind that change?

Reply:
It just makes testing a lot easier. I thought it wasn't a very invasive change so it was worth doing. I suppose I could try and do it with mocking or something instead but it is kind of useful to be able to specify the artifact name,